### PR TITLE
Fix labels for CompleteOptionsInverted

### PR DESCRIPTION
**Pull request by anotherminh**
## Related Issues

- Closes https://transcend.height.app/T-32229

## Security Implications

_[none]_

## System Availability

_[none]_

### CODE REVIEW

**Review by MadDataScience — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -75,4 +75,4 @@
     "typescript": "^4.7.4"
   },
   "packageManager": "yarn@3.4.1"
-}
+}
```
Comment (MadDataScience) [inline package.json:78]: Oops, looks like your linter removed the ending newline

File: src/components/constants.ts [hunk 2]
```diff
@@ -10,5 +13,16 @@ export const DEFAULT_PURPOSE_TO_MESSAGE_KEY: Record<string, DefinedMessage> = {
   SaleOfInfo: completeOptionsMessages.saleOfInfoLabel,
 };
 
+export const DEFAULT_PURPOSE_TO_INVERTED_MESSAGE_KEY: Record<
+  string,
+  DefinedMessage
+> = {
+  Essential: completeOptionsMessages.essentialLabel,
```
Comment (eligrey) [inline src/components/constants.ts:20]: should this be

```suggestion
  Essential: completeOptionsInvertedMessages.essentialLabel,
```

Comment (anotherminh) [reply src/components/constants.ts:20]: no that's not a thing

**Review by eligrey — APPROVED** [review]

File: src/components/constants.ts (hunk 2 above)
Comment (anotherminh) [reply src/components/constants.ts:20]: (i know it looks like it should be, but I checked)

**Pull request merged**